CMakeLists.txt

```diff
@@ -58,9 +58,9 @@ add_dependencies(vclient UpdateVersion)
 add_executable(vsim ${vsim_SRCS})
 add_dependencies(vsim UpdateVersion)
 
+add_subdirectory(${CMAKE_CURRENT_SOURCE_DIR}/doc/man)
```

Comment:
Sure you want to add this as a subdirectory and not directly here? I would add it to the main `CMakeLists.txt` as it's not too much code. Would be still be overseeable imo.

Reply:
IMO this modularity comes at no cost in this case and makes changes to the docs clearly separateable from the source codes.

Reply:
Let every man seek heaven in his own fashion ;-)